### ISSUE TITLE
Pass EventTooltip feature template method in the template parameters

### ISSUE THREAD

**Issue by ghulamghousdev** [bryntum/support#8419]
https://bryntum.com/products/scheduler/docs/api/Scheduler/feature/EventTooltip#config-template

Currently we need to do something like this to get the Feature template method: `this.$meta.config.template`

**Comment by ExtAnimal** [comment]
Yes, we could add a `super` or `default` property into the data which is a reference to the default template.
